sources/packages/backend/apps/api/src/route-controllers/application/_tests_/application.students.getCompletedApplicationDetails.e2e-spec.ts

```diff
@@ -0,0 +1,233 @@
+    // have the scholasticStandingChangeType returned.
+    scholasticStanding.application = application;
+    await studentScholasticStandingRepo.save(scholasticStanding);
+
+    const endpoint = `/students/application/${application.id}/completed`;
+    const token = await getStudentToken(
+      FakeStudentUsersTypes.FakeStudentUserType1,
+    );
+    // Act/Assert
+    await request(app.getHttpServer())
+      .get(endpoint)
+      .auth(token, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.OK)
+      .expect({
+        firstDisbursement: {
+          coeStatus: firstDisbursement.coeStatus,
+          disbursementScheduleStatus:
+            firstDisbursement.disbursementScheduleStatus,
+        },
+        assessmentTriggerType: application.currentAssessment.triggerType,
+        scholasticStandingChangeType:
+          StudentScholasticStandingChangeType.StudentDidNotCompleteProgram,
+      });
+  });
+
+  it(`Should get application details with the most updated appeal status when the application has more than one student appeals associated with.`, async () => {
+    // Arrange
+    const application = await saveFakeApplicationDisbursements(appDataSource, {
+      student,
+    });
+    application.applicationStatus = ApplicationStatus.Completed;
+    await applicationRepo.save(application);
+    const [firstDisbursement] =
+      application.currentAssessment.disbursementSchedules;
+    firstDisbursement.coeStatus = COEStatus.completed;
+    await disbursementScheduleRepo.save(firstDisbursement);
+    // Create approved student appeal
+    const approvedAppealRequest = createFakeStudentAppealRequest();
+    approvedAppealRequest.appealStatus = StudentAppealStatus.Approved;
+    const approvedAppeal = createFakeStudentAppeal({
+      application,
+      appealRequests: [approvedAppealRequest],
+    });
+    approvedAppeal.submittedDate = addDays(-1);
+    // Create declined student appeal from yesterday
+    const pendingAppealRequest = createFakeStudentAppealRequest();
+    pendingAppealRequest.appealStatus = StudentAppealStatus.Pending;
+    const pendingAppeal = createFakeStudentAppeal({
+      application,
+      appealRequests: [pendingAppealRequest],
+    });
+    await studentAppealRepo.save([approvedAppeal, pendingAppeal]);
+
+    const endpoint = `/students/application/${application.id}/completed`;
+    const token = await getStudentToken(
+      FakeStudentUsersTypes.FakeStudentUserType1,
+    );
+    // Act/Assert
+    await request(app.getHttpServer())
+      .get(endpoint)
+      .auth(token, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.OK)
+      .expect({
+        firstDisbursement: {
+          coeStatus: firstDisbursement.coeStatus,
+          disbursementScheduleStatus:
+            firstDisbursement.disbursementScheduleStatus,
+        },
+        assessmentTriggerType: application.currentAssessment.triggerType,
+        appealStatus: StudentAppealStatus.Pending,
```

Comment:
okay,  so, I think it's the comment issue (line 201) https://github.com/bcgov/SIMS/pull/1832/files#r1147982083. latest is pending according to logic